app/src/main/java/org/p2p/wallet/auth/AuthModule.kt

```diff
@@ -38,6 +42,11 @@ object AuthModule {
     fun create() = module {
         single { BiometricManager.from(androidContext()) }
 
+        factory { GoogleSignInHelper() }
+        factory { WalletWeb3AuthManager(get(), get(), get(), get()) }
+
+        factory { NewOnboardingPresenter(get()) } bind NewOnboardingContract.Presenter::class
```

Comment:
i would extract it as an extension Module.newOnboardingScreensModule inside AuthModule